<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HASHClass</class>
 <widget class="QMainWindow" name="HASHClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HASH</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-top-color: rgb(170, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">
        #centralWidget{border-image: url(:/HASH/Resources/images/nanyou.jpg);}
</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>711</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPlainTextEdit" name="input_plaintext">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>551</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 200);
font: 9pt &quot;Consolas&quot;;</string>
      </property>
      <property name="plainText">
       <string>请输入明文...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Butn_compute">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>50</y>
        <width>101</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;Consolas&quot;;
background-color: rgba(255, 255, 255, 200);</string>
      </property>
      <property name="text">
       <string>MD5</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="input_plaintext_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>551</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 200);
font: 9pt &quot;Consolas&quot;;</string>
      </property>
      <property name="plainText">
       <string>请输入明文...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_computeSHA1">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>310</y>
        <width>101</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;Consolas&quot;;
background-color: rgba(255, 255, 255, 200);</string>
      </property>
      <property name="text">
       <string>SHA1</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit_MD5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>551</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 200);
font: 9pt &quot;Consolas&quot;;</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit_SHA1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>460</y>
        <width>551</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 200);
font: 9pt &quot;Consolas&quot;;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_openfile1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 200);</string>
      </property>
      <property name="text">
       <string>打开文件</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_openfile2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 200);</string>
      </property>
      <property name="text">
       <string>打开文件</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>114</width>
      <height>478</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="Select_Button_plain">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;楷体&quot;;
background-color: rgba(255, 255, 255, 200);
font: 10pt &quot;Agency FB&quot;;</string>
       </property>
       <property name="text">
        <string>明文散列</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Select_Button_file">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;楷体&quot;;
background-color: rgba(255, 255, 255, 200);
font: 10pt &quot;Agency FB&quot;;</string>
       </property>
       <property name="text">
        <string>文件签名</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>109</width>
         <height>388</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>853</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="HASH.qrc"/>
 </resources>
 <connections/>
</ui>
